src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE = 40;
+  const LONG_TERMIN = 7;
+  const SHORT_TERMIN = 3;
```

Comment:
Similarly, `SHORT_TERMIN` should be renamed to `SHORT_TERM` to be descriptive and align with standard terminology.